.golangci.yml

```
@@ -0,0 +1,6 @@
+linters:
+  enable:
```

```suggestion
  disable-all: true
  enable:
```

Do we need this here, it's in the network operator's `.golangci.yml`?